playbooks/Docker-Seleinum/test_vault.yml

```diff
@@ -0,0 +1,6 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
I don't think its critical, but probably not good practice to commit this.

Reply:
It is all encrypted, is that still not ok to commit?

Reply:
Which is why I said its not critical. I don't know the details about how Ansible implements the encryption. Depending on the exact details it may be brute forceable. I have seen some projects that actually recommend committing encrypted credentials (https://github.com/AGWA/git-crypt).

I'm also assuming this is useless without the encryption key (which should 100% not be commited), which makes it similar to a local file (like .DS_STORE or .vscode).

Reply:
You are correct, you can't do anything with this file without the key, but I'll remember this for next time.